src/extract.jl

```diff
@@ -587,7 +587,7 @@ function read_standard_header(
         check_checksum_field(buf)
     catch err
         if err isa ErrorException
```

Comment:
Unrelated to the PR but this should probably not be a standard `ErrorException` but something more specific.